src/Riok.Mapperly/Descriptors/MappingBuilders/EnumToStringMappingBuilder.cs

```diff
@@ -26,11 +28,56 @@ public static class EnumToStringMappingBuilder
 
     private static EnumToStringMapping BuildEnumToStringMapping(MappingBuilderContext ctx)
     {
-        var ignoredSourceMembers = ctx.Configuration.Enum.IgnoredSourceMembers.ToHashSet(SymbolTypeEqualityComparer.FieldDefault);
-        var explicitMappings = BuildExplicitValueMappings(ctx);
+        var enumMemberMappings = BuildEnumMemberMappings(ctx);
+        var fallbackMapping = BuildFallbackMapping(ctx);
+
+        if (fallbackMapping.FallbackExpression is LiteralExpressionSyntax fallbackLiteral)
+        {
+            enumMemberMappings = enumMemberMappings.Where(IsNotEquivalentTo(fallbackLiteral));
+        }
```

Comment:
Similar to the other comment:
Instead of comparing the generated syntax tree here I think it would make more sense to 
* call `BuildFallbackParseMapping` before `BuildEnumMemberMappings`
* add a `out string? fallbackStringValue` parameter to `BuildFallbackMapping`
* pass this `fallbackStringValue` to `BuildEnumMemberMappings` and before returning an `EnumMemberMapping` continue/skip the field if its target string value is equal to the passed `fallbackStringValue`

Reply:
Agreed, implemented the suggested changes.